odoo/addons/base/models/ir_module.py

```diff
@@ -117,7 +117,12 @@ class MyFilterMessages(Transform):
     default_priority = 870
 
     def apply(self):
-        for node in self.document.traverse(nodes.system_message):
+        if hasattr(self.document, 'findall'):
```

Comment:
Could you add a small comment to mention it is because of different lib versions?